### PR TITLE
DOCSP-38331 Add Alternative Compass Import Export Note

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Add a callout to the mongo dump import and export page that you can also import and export with Compass.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-38331/mongoimport/

## JIRA

https://jira.mongodb.org/browse/DOCSP-38331

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=661d8fbf3a5920a29170ecf1

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/mongoimport.txt [hunk 1]
```diff
@@ -41,13 +41,16 @@ tool.
 Run ``mongoimport`` from the system command line, not the 
 :binary:`~bin.mongo` shell.
 
-.. seealso::
-
-   :binary:`~bin.mongoexport` which provides the corresponding
-   structured data export capability.
+:binary:`~bin.mongoexport` provides the corresponding
```
Comment (jmd-mongo) [inline source/mongoimport.txt:44]: [suggestion]

I realize this copy existed prior to this PR, but I think we should consider simplifying this statement.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for this update, @ianf-mongodb! I left a suggestion for your consideration.

Thanks!
Joe

**Comment by jaoppenheim** [comment]
LGTM

File: source/mongoimport.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/mongoimport.txt]: Sure I updated to a similar but simpler:

```
:binary:`~bin.mongoexport` provides data export capabilities.
```

**Review by ianf-mongodb — COMMENTED** [review]
Thanks @jmd-mongo I have updated the previous copy regarding `mongoexport`.

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update, @ianf-mongodb. This one LGTM at this point.

Thanks,
Joe

**Pull request merged**